crates/winscard/src/winscard.rs

```diff
@@ -250,6 +413,118 @@ pub struct TransmitOutData {
     pub receive_pci: Option<IoRequest>,
 }
 
+/// This structure represents the result of the `SCardConnect` function.
+pub struct ScardConnectData {
+    /// Established smart card handle.
+    pub scard: Box<dyn WinScard>,
```

Comment:
naming: Should it be named `handle`? I see things like `scard.scard` at some places.

```suggestion
    pub handle: Box<dyn WinScard>,
```

Reply:
agreed. done